package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless

```diff
@@ -20,6 +21,15 @@ local function get_channel(radio, config)
 	return channel or config.channel()
 end
 
+local function get_htmode(radio)
+       local phy = util.find_phy(uci:get_all('wireless', radio))
```

Comment:
Hmm, I will look into getting rid of this `get_all()` by changing `iterate_radios()`.

Reply:
`radio` is the full UCI section now, please rebase.